content/post/deploy-mssql-server-in-aks/index.md

```diff
@@ -0,0 +1,454 @@
+NAME                                   TYPE                       DATA   AGE
+mssqlserver-ag-cluster-auth            kubernetes.io/basic-auth   2      9m36s
+mssqlserver-ag-cluster-client-cert     kubernetes.io/tls          3      9m36s
+mssqlserver-ag-cluster-dbm-login       kubernetes.io/basic-auth   1      9m36s
+mssqlserver-ag-cluster-endpoint-cert   kubernetes.io/tls          3      9m36s
+mssqlserver-ag-cluster-master-key      kubernetes.io/basic-auth   1      9m36s
+mssqlserver-ag-cluster-server-cert     kubernetes.io/tls          3      9m36s
+
+$ kubectl get service -n demo -l=app.kubernetes.io/instance=mssqlserver-ag-cluster 
+NAME                               TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)    AGE
+mssqlserver-ag-cluster             ClusterIP   10.128.113.192   <none>        1433/TCP   9m43s
+mssqlserver-ag-cluster-pods        ClusterIP   None             <none>        1433/TCP   9m43s
+mssqlserver-ag-cluster-secondary   ClusterIP   10.128.213.243   <none>        1433/TCP   9m43s
+```
+
+From the above list, the `mssqlserver-ag-cluster-auth` Secret contains the admin-level credentials needed to connect to the database. Use the following commands to obtain the username and password:
+
+```bash
+$ kubectl get secret -n demo mssqlserver-ag-cluster-auth  -o jsonpath='{.data.username}' | base64 -d
+sa
+$ kubectl get secret -n demo mssqlserver-ag-cluster-auth  -o jsonpath='{.data.password}' | base64 -d
+pT42K58uKEe3pote
+```
+
+
+### Insert Sample Data
+
+In this section, we are going to login into our MSSQL Server pod and insert some sample data.
+
+```bash
+$ kubectl exec -it mssqlserver-ag-cluster-0 -n demo bash
+Defaulted container "mssql" out of: mssql, mssql-coordinator, mssql-init (init)
+mssql@mssqlserver-ag-cluster-0:/$ /opt/mssql-tools/bin/sqlcmd -S localhost -U sa -P "pT42K58uKEe3pote"
+
+1> SELECT name FROM sys.databases
+2> GO
+name                                                                                                                            
+--------------------------------------------------------------------------------------------------------------------------------
+master                                                                                                                          
+tempdb                                                                                                                          
+model                                                                                                                           
+msdb                                                                                                                            
+music                                                                                                                           
+kubedb_system                                                                                                                   
+
+(6 rows affected)
+1> USE music
```

Comment:
Before using this `music` database, we can show that is database is added in the availability group. 
Example
1> SELECT database_name 
2> FROM sys.availability_databases_cluster
3> go
database_name                                                                                                                   

agdb1                                                                                                                           
agdb2